clustermesh/clustermesh.go

```diff
@@ -905,6 +905,16 @@ func (k *K8sClusterMesh) Connect(ctx context.Context) error {
 			aiLocal.ClusterName, aiLocal.ClusterID)
 	}
 
+	cid, err := strconv.Atoi(aiRemote.ClusterID)
+	if err != nil {
+		k.Log("❌ Cluster has the non-numeric ID 0 (cluster ID: %s): %s", aiLocal.ClusterID, err)
+		return err
```

Comment:
I think we should return the above log message as an error instead. The strconv.Atoi error itself is not very useful to the user.